docs/structured-streaming-programming-guide.md

```diff
@@ -2320,6 +2320,11 @@ Here are the configs regarding to RocksDB instance of the state store provider:
     <td>Whether we perform a range compaction of RocksDB instance for commit operation</td>
     <td>False</td>
   </tr>
+  <tr>
+    <td>spark.sql.streaming.stateStore.rocksdb.changelogCheckpointing.enabled</td>
+    <td>Whether to upload changelog instead of snapshot during RocksDB StateStore commit</td>
```

Comment:
Nit: `instead of full snapshot during RocksDB state store commit`

Reply:
It is a matter of wording,  but we didn't upload full snapshot even without changelog checkpointing, sst files were reused across versions.

Reply:
Add a separate section for changelog checkpointing.